app/src/main/java/com/android/shelfLife/ui/utils/UIComponents.kt

```diff
@@ -93,37 +95,62 @@ fun CustomTopAppBar(
 fun CustomSearchBar(
     query: String,
     onQueryChange: (String) -> Unit,
+    onDeleteTextClicked: () -> Unit,
```

Comment:
I like that you have left onDeleteTextClicked as a parameter as it allows us to reuse this function in different situations